authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.1.7 on 2019-03-07 10:46
```

Comment:
@kelvinndmo Remove this migrations file from the PR.

Reply:
Migration file removed. Thanks for picking out the error